book/src/usage.md

```diff
@@ -7,3 +7,27 @@ can be accessed via the command `hx --tutor` or `:tutor`.
 > 💡 Currently, not all functionality is fully documented, please refer to the
 > [key mappings](./keymap.md) list.
 
+## Modes
+
+Helix is a modal editor, meaning it has different modes for different tasks. The main modes are:
+
+* [Normal mode](./keymap.md#normal-mode): For navigation and editing commands. This is the default mode.
+* [Insert mode](./keymap.md#insert-mode): For typing text directly into the document. Access by typing `i` in normal mode.
+* [Select/extend mode](./keymap.md#select--extend-mode): For making selections and performing operations on them. Access by typing `v` in normal mode.
+
+## Buffers
+
+Buffers are in-memory representations of files. You can have multiple buffers open at once, allowing you to work on several files without closing them. Use pickers (see below) to switch between open buffers or commands like `:buffer-next` and `:buffer-previous`.
+
+## Selection-first editing
+
+Inspired by [Kakoune](http://kakoune.org/), Helix follows the `selection → action` model. This means that whatever you are going to act on (a word, a paragraph, a line, etc.) is selected first and the action itself (delete, change, yank, etc.) comes second. A cursor is simply a single width selection.
```

Comment:
This is copied straight from the Migrating from Vim page, but that seems ok to me.

https://github.com/helix-editor/helix/blob/f65ec32a1c2e09b3b32b521617f4a3ef19bc71c5/book/src/from-vim.md#L1-L12